include/fmt/compile.h

```diff
@@ -11,7 +11,12 @@
 #include "format.h"
 
 FMT_BEGIN_NAMESPACE
-namespace detail {
+FMT_MODULE_EXPORT_BEGIN
+
+// A compile-time string which is compiled into fast formatting code.
+class compiled_string {};
```

Comment:
Does it still need to be public if we don't export the `FMT_COMPILE` macro and only provide a UDL?

Reply:
Well spotted - thanks!
All related stuff removed.